web-ui/src/app/components/latest-blocks/latest-blocks.component.ts

```diff
@@ -70,9 +72,30 @@ export class LatestBlocksComponent implements OnInit, OnDestroy {
   }
 
   private onBlockRetrieved(response: Block[]) {
+    this.countNewestBlocks(response);
+    this.latestBlock = response[0];
     this.blocks = response;
   }
 
+  private countNewestBlocks(newBlocks: Block[]) {
```

Comment:
I would avoid functions with side effects as much as possible, as a example, you mutating the `totalNewestBlocks`, instead you can return the number of new blocks and do whatever you want with the number, think about which version would be simpler to test.